redirects.txt

```diff
@@ -471,7 +471,7 @@ cloud-images/introduction reference/clouds/find-cloud-images/
 ubuntu-ha-migrate-from-crmsh-to-pcs reference/high-availability/migrate-from-crmsh-to-pcs/
 migrate-from-crmsh-to-pcs reference/high-availability/migrate-from-crmsh-to-pcs/
 
-basic-backup-shell-script reference/backups/basic-backup-shell-script/
+# basic-backup-shell-script reference/backups/basic-backup-shell-script/
```

Comment:
```suggestion
basic-backup-shell-script reference/backups/basic-backup-shell-script/
```

Reply:
This is an old redirect that used to exist in Discourse, which we need to keep. The "to" path is correctly pointing back to Discourse, but we can't take it out because otherwise users on the old path will 404, and I want to send them back to the Discourse post (it's a very confusing circle, but necessary to maintain)